### PR TITLE
utm*: normalize docs

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Normalize doc blocks.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/modules/utm_aaa_group.py
plugins/modules/utm_aaa_group_info.py
plugins/modules/utm_ca_host_key_cert.py
plugins/modules/utm_ca_host_key_cert_info.py
plugins/modules/utm_dns_host.py
plugins/modules/utm_network_interface_address.py
plugins/modules/utm_network_interface_address_info.py
plugins/modules/utm_proxy_auth_profile.py
plugins/modules/utm_proxy_exception.py
plugins/modules/utm_proxy_frontend.py
plugins/modules/utm_proxy_frontend_info.py
plugins/modules/utm_proxy_location.py
plugins/modules/utm_proxy_location_info.py

### CODE REVIEW

File: plugins/modules/utm_aaa_group.py [hunk 1]
```diff
@@ -8,120 +8,117 @@
...
+    default: ''
+  members:
+    description:
+      - A list of user ref names (aaa/user).
+    type: list
+    elements: str
+    default: []
+  network:
+    description:
+      - The network reference name. The objects contains the known ip addresses for the authentication object (network/aaa).
```
Comment (felixfontein) [inline plugins/modules/utm_aaa_group.py:100]:
```suggestion
      - The network reference name. The objects contains the known IP addresses for the authentication object (network/aaa).
```

**Review by felixfontein — APPROVED** [review]

File: plugins/modules/utm_aaa_group.py (hunk 1 above)
Comment (russoz) [reply plugins/modules/utm_aaa_group.py:100]: There are many improvements to be made in those docs, but manually.

Comment (felixfontein) [reply plugins/modules/utm_aaa_group.py]: True :) I'm adding suggestions for ones that I notice while skimming over the changes, so we can already fix some of them now.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**